docs/operator-guide/upgrade-edp-3.8.md

```diff
@@ -0,0 +1,174 @@
+# Upgrade EDP v3.7 to 3.8
+
+!!! Important
+    We suggest backing up the EDP environment before starting the upgrade procedure.
+
+This section provides detailed instructions for upgrading the EPAM Delivery Platform to version 3.8.1. Follow the steps and requirements outlined below:
+
+
+1. Delete the following resources:
+
+    ```bash
+    kubectl -n edp delete ingress edp-headlamp
+    kubectl -n edp delete ingress edp-tekton-dashboard
+    kubectl -n edp delete ingress el-github-listener
+    kubectl -n edp delete EDPComponents sonar
+    kubectl -n edp delete EDPComponents nexus
+    kubectl -n edp delete EDPComponents argocd
+    kubectl -n edp delete EDPComponents main-keycloak
+    kubectl -n edp delete EDPComponents tekton
+    kubectl -n edp delete EDPComponents github
+    ```
+
+2. Delete the following resources CRD:
+
+    ```bash
+    kubectl delete CustomResourceDefinition edpcomponents.v1.edp.epam.com
+    ```
+
+3. Update EDP Portal url, in Keycloak:
+
+    Navigate to Keycloak realm with `Portal` client. Update `Valid redirect URIs` field:
+
+    ```text
+    Old value: https://edp-headlamp-edp.<dns_wildcard>/*
+
+    New value: https://portal-edp.<dns_wildcard>/*
+    ```
+
+    !![Keycloak client](../assets/operator-guide/portal-keycloak.png "Keycloak client")
+
```

Comment:
Please add description that this KubeRocketCI version require ArgoCD version with applicationset support and provide link to our guide with argocd RBAC hack.